inst/rmarkdown/templates/vetiver_dashboard/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,49 @@
+---
+title: "Monitor model"
+output: 
+  vetiver::vetiver_dashboard:
+    board: 'pins::board_rsconnect()'
```

Comment:
Just to share that this will be passed as a string to the function. So `board` will be equal to `"pins::board_rsconnect()"` inside your format function `vetiver_dashboard`. 

This means that `board$versioned` will probably error. I think you need to evaluate the expression that you are passing as a string through YAML. 

Otherwise you could pass evaluated R expression but this would require `!expr`

````yaml
output: 
  vetiver::vetiver_dashboard:
    board: !expr pins::board_rsconnect()
````

And maybe it would create other issue 🤔 

**bslib** leverages that when you want to pass a `bs_theme()` object using YAML
https://rstudio.github.io/bslib/articles/bslib.html#low-level
It was the only possibility at the beginning but this has been simplified so that YAML key value are passed as argument to `bs_theme` instead

````yaml
output:
  html_document:
    theme:
      bg: "#002B36" 
      fg: "#EEE8D5"
      progress-bar-bg: "orange"
````

equivalent to
````yaml
output:
    html_document:
        theme: !expr bs_theme(bg = "#002B36", fg = "#EEE8D5",  "progress-bar-bg" = "orange")
)
````

So you could imagine something like 

````yaml
output: 
  vetiver::vetiver_dashboard:
    board: 
        board_rsconnect
               auth: envvar
              version: FALSE
```` 

Just some thoughts as I was looking at your PR for your initial rmarkdown question